src/components/SharedComponents/Buttons/RotatingINatIconButton.tsx

```diff
@@ -112,7 +109,6 @@ const RotatingINatIconButton = ( {
         icon={icon}
         mode={mode}
         onPress={onPress}
-        rotating={rotating}
```

Comment:
This prop does not exist on the `INatIconButton` component

Reply:
Correct. Seems good to be removed